.github/actions/cancel-workflow/action.yml

```diff
@@ -1,7 +1,13 @@
 name: Cancel Workflow
 description: >
   Cancels the current workflow run, i.e. all jobs. Useful if you want to cancel the rest of the workflow when one job
-  fails.
+  fails. Note that this will cause the workflow to appear cancelled, not failed.
+
+# Cancelling the workflow in a post-script (like this:
+# https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#runspost; can also be done with
+# this action: https://github.com/webiny/action-post-run, see Git history of this file) wouldn't help the status, it
+# would still be cancelled. It actually indeed is, but it would be nicer to set it to failed, but there seems to be no
+# way to do this.
```

Comment:
I had some fun experimenting around with this without avail.

But at least I had fun.

![image](https://user-images.githubusercontent.com/1976647/202059944-5579e032-049f-4920-a61a-22a76d9d626c.png)